### PR TITLE
MUItflix - project express api by Olof

### PR DESCRIPTION
**Pull request by luuf81**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -27,6 +21,70 @@ app.get('/', (req, res) => {
...
+
+//show with a specific title
+
+app.get('/shows/title/:title', (req, res) => {
+  const titleID = req.params.title
+  const titleWithID = data.find(item => item.show_id == +titleID)
+  console.log(titleWithID)
+  if(!titleWithID) res.send('Not Found!!')
+  else res.json(titleWithID)
+})
```
Comment (HIPPIEKICK) [inline server.js:86]: You've named the param title but expect an ID? 🤔 Change the naming here so it adds up :)

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
Good job Olof - and really nice with the frontend! Please change your naming so that the endpoint that returns a single result makes more sense - `/shows/:showID` maybe? Remove `console.log`s and add some curly brackets to your if/else-statements.

**Comment by luuf81** [comment]
Fixed changes required by Matilda (naming of single endpoints, console.logs, curly brackets)

File: server.js (hunk 1 above)
Comment (luuf81) [reply server.js]: Done

File: server.js [hunk 2]
```diff
@@ -27,6 +21,70 @@ app.get('/', (req, res) => {
   res.send('Hello world')
```
[the remark this answers is not in the document]
Comment (luuf81) [reply server.js]: Thanks! I will use this one from now on :)

**Review by KarolCislowski — APPROVED** [review]
Great job!

**Pull request closed without merging**